src/react-app.js

```diff
@@ -956,7 +963,7 @@ function renderReactApp({
             <Route exact path="/">
               <Home mode={mode} setMode={setMode} />
             </Route>
-            <Route exact path="/:exerciseNumber">
+            <Route exact path="/:exerciseNumber(\\d+)">
```

Comment:
Is this correct? I tried on the example deploy and this doesn't appear to work properly:

![image](https://user-images.githubusercontent.com/1500684/96515707-057a7980-1223-11eb-8385-b5f3fd56f471.png)

Reply:
Well, apart from it not working at all, this is going pretty well! Thanks for the review, I didn't spot there's a preview deploy so that's really useful.

I'll find out what's happening..

Reply:
@kentcdodds that should be fixed now, preview site now looks ok